python-service/src/app.py

```
@@ -61,5 +103,25 @@ def review_function():
 
 
+@app.route("/cache/clear", methods=["POST"])
+def clear_cache():
+    cache.clear()
+    return jsonify({"message": "Cache cleared successfully"})
+
+
+@app.route("/cache/stats", methods=["GET"])
+def cache_stats():
+    active_entries = sum(1 for entry in cache.values()
+                        if time.time() < entry["expires_at"])
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)